### PR TITLE
Fix installation description for Apple Silicon #2955

### PR DESCRIPTION
**Pull request by stultiloquence**
# Fix installation description for Apple Silicon

Fixes Issue: #2955

### CODE REVIEW

**Review by jannikr — APPROVED** [review]

**Review by BridgeAR — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -59,21 +59,7 @@
...
-```bash
-# Clone the CodeCharta repo
-$ git clone https://github.com/MaibornWolff/codecharta.git
-# Navigate to Visualization
-$ cd visualization
-# Run the development server
-$ npm run dev
-# Upload any .cc.json!
-```
+-   **Local:** See [here](visualization/readme.md).
```
Comment (BridgeAR) [inline README.md:62]:
```suggestion
-   **Local:** See [here](visualization/README.md).
```

**Review by BridgeAR — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (BridgeAR) [reply README.md:62]: Linux is not lenient when it comes to caps / no caps.

**Review by ce-bo — APPROVED** [review]
LGTM

**Comment by jannikr** [comment]
Merge with branch protection bypass due to a missing Sonar token in the forked repository. It should be possible to merge successfully as there are only changes in the Readme.md files.

**Pull request merged**